docs/scripts/fetch_contributors.py

```
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)